countrycodepicker/src/main/java/com/rilixtech/widget/countrycodepicker/CountryUtils.java

```diff
@@ -125,6 +125,8 @@ public static int getFlagDrawableResId(Country country) {
         return R.drawable.flag_croatia;
       case "cu": //cuba
         return R.drawable.flag_cuba;
+        case "cw": 
```

Comment:
oops, I only thought to format this file (according to the current setting of 2 spaces for tabs), looks like you formatted `CountryCodeArrayAdapter` instead. Can you format both with 2 spaces so they only show your changes in a way that's consistent with the existing codebase? - we will want to offer this feature to the parent repo

Reply:
yes, will apply safe check in CountryCodeArrayAdapter as well.